linera-storage-service/src/server.rs

```diff
@@ -279,7 +279,10 @@ async fn main() {
                 path_buf,
                 common_config,
             };
-            let (store, _) = RocksDbStore::new(config).await.expect("store");
+            let namespace = "linera";
+            let store = RocksDbStore::initialize(&config, namespace)
```

Comment:
In line with my other comments, this "initialize" is a little suspicious.

Reply:
I think it is in line with previous use. If one exists we use it, if not we create it.